### DISCUSSION TITLE
replicating example of rel_component

### DISCUSSION THREAD

**Discussion by staedi** [explosion/spaCy#10881] [Help: Other Questions] [answered]
Hello,

I'm currently trying to learn and apply `rel_component` for general usage.
I found the example output is pretty close to my intention. (Given entities, extract actions between them)

However, since the dataset used in the tutorial is medical data, I get that the result would not be directly applied to more general case.

If I want to replicate the example, is modifying `parse_data.py` enough [SYMM_LABELS, MAP_LABELS]?

```
**doc = nlp("Amsterdam is the capital of the Netherlands.")**
print("spans", [(e.start, e.text, e.label_) for e in doc.ents])
for value, rel_dict in doc._.rel.items():
    print(f"{value}: {rel_dict}")

# spans [(0, 'Amsterdam', 'LOC'), (6, 'Netherlands', 'LOC')]
# (0, 6): {'CAPITAL_OF': 0.89, 'LOCATED_IN': 0.75, 'UNRELATED': 0.002}
# (6, 0): {'CAPITAL_OF': 0.01, 'LOCATED_IN': 0.13, 'UNRELATED': 0.017}
```

Thanks!

**Comment by polm** [comment, marked as answer]
Here's a link to the project for reference.

https://github.com/explosion/projects/tree/v3/tutorials/rel_component

I think those changes should be enough. The important thing is just the structure of data in the Doc, in this case the contents of `doc._.rel`. If you set that the same way it is in the example project, even if the labels are different, it should be fine.
